charts/servicebusexporter/templates/deployment.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "servicebusexporter.fullname" . }}
+  labels:
+{{ include "servicebusexporter.labels" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
```

Comment:
It makes little sense to use more than one replica, since all of them will be scraped separately. That will result in multiple copies of the metrics and they will be difficult to separate.